### ISSUE TITLE
Can it Works on UWP?

### ISSUE THREAD

**Issue by iCCat001** [SuavePirate/Xamarin.Forms.Controls.FloatingActionButton#1]
Hi, I try this in my Xamarin.Forms Project, It's very Good. But,It only work on Android & iOS, How to make it works on UWP? Because that I an not very good at XAML, I have no idea with this.

**Comment by SuavePirate** [comment]
It's on my list of things to do. If you try to use it in UWP, it will render a regular button for now.

**Comment by Jatsai** [comment]
Does it work on UWP now?

**Comment by SuavePirate** [comment]
Yeah, it still isn't on UWP yet. I'd love to have some help implementing it since my hands are full with other projects at the moment. I'm always accepting pull requests.
